src/cruds.js

```diff
@@ -0,0 +1,49 @@
+let arrayobj = [];
+export const addlist = (value) => {
```

Comment:
- [ ] I can't add a task to the list or local storage, as the previous reviewer mentioned kindly make sure to fix this so your app works properly

![Peek 2022-10-15 16-22](https://user-images.githubusercontent.com/25227022/195994610-45cfc242-f45f-4e09-ad87-d20ba25f0b6c.gif)